opentracing-api/src/main/java/io/opentracing/ActiveSpanSource.java

```diff
@@ -24,6 +24,10 @@
     /**
      * @return the {@link ActiveSpan active span}, or null if none could be found. This does not affect the reference
      * count for the {@link ActiveSpan}.
+     *
+     * <p>
```

Comment:
Is this paragraph intended as method documentation or addition to the `@return` statement?
Normally `@return` is below the method documentation.